src/datetime.js

```diff
@@ -155,11 +155,12 @@ function parseDataToDateTime(parsed, parsedZone, opts, format, text) {
   const { setZone, zone } = opts;
   if (parsed && Object.keys(parsed).length !== 0) {
     const interpretationZone = parsedZone || zone,
-      inst = DateTime.fromObject(
-        Object.assign(parsed, opts, {
-          zone: interpretationZone
-        })
-      );
+      obj = Object.assign(parsed, opts, {
+        zone: interpretationZone
+      });
+    // setZone is a valid option in calling methods, but not in fromObject
+    delete obj.setZone;
```

Comment:
would be a little nicer to just put `setZone: null` in the `assign` call just above

Reply:
Done. That was not an option with a previous version of my code.